kotlin/README.md

```diff
@@ -0,0 +1,137 @@
+## Quick Example for a Kotlin Worker (3 minutes)
+
+This example will show you how to build your Kotlin IronWorker, test it locally, then upload it
+to IronWorker for production.
+
+**Note**: Be sure you've followed the base [getting started instructions on the top level README](https://github.com/iron-io/dockerworker).
+
+### 1. Build including the dependencies:
+
+```sh
+docker run --rm -v "$PWD":/worker -w /worker iron/kotlin kotlinc -cp "json-java.jar:gson-2.2.4.jar:ironworker-1.0.10.jar" Hello.kt PayloadData.kt
+```
+
+### 2. Test locally
+
+Now run it to test it out:
+
+```sh
+docker run -e PAYLOAD_FILE=hello.payload.json --rm -v "$PWD":/worker -w /worker iron/kotlin kotlin -cp gson-2.2.4.jar:json-java.jar:ironworker-1.0.10.jar:. Hello
```

Comment:
Could you change the run to use the JRE image and the java executable? Again, I know nothing about this, so please excuse me if that is not possible.